client/components/sorted-grid/index.jsx

```diff
@@ -29,7 +28,7 @@ class SortedGrid extends PureComponent {
 		const items = [];
 
 		for ( let i = 0; i < this.props.items.length; i += this.props.itemsPerRow ) {
-			const row = slice( this.props.items, i, i + this.props.itemsPerRow );
+			const row = this.props.items.slice( i, i + this.props.itemsPerRow );
```

Comment:
I don't think a guard here will be necessary - we're inside a loop that already uses `this.props.items` as an array because it accesses its `.length`, so I think we're already safe.

Reply:
Oh gosh, of course, I totally missed that, thanks!